builder/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN dnf -y update \
         bison \
         ca-certificates \
         clang \
+        clang-tools-extra \
```

Comment:
I don't know if we want to add this package to the builder, it is pretty heavy. We might want to start considering creating a dev image that is based on but separate from the builder image, that way we can have a slimmer builder for CI to use.

Reply:
I will keep this change locally for now